cli/src/cli.ts

```diff
@@ -0,0 +1,53 @@
+import { Command } from '@commander-js/extra-typings';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import XMLHttpRequest from 'xhr2';
+
+import { version } from '../package.json';
+
+import { handleUploadCommand } from './commands/upload';
+import { ENVIRONMENT } from './env';
+import { logErrorMessage } from './utils/log';
+
+/*
+  We use the msadoc-client to perform HTTP requests.
+  This library internally depends on XMLHttpRequest, which is unfortunately not available in NodeJS.
+  (When trying to perform a request to the server, you get "ReferenceError: XMLHttpRequest is not defined".)
+  Thus, we need to add a custom library for this.
+*/
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+global.XMLHttpRequest = XMLHttpRequest;
```

Comment:
This was kinda ugly: Our `msadoc-client` uses `rxjs/ajax` to perform http requests. Unfortunately, `rxjs/ajax` internally uses `XMLHttpRequest`, which is an API that is only available in the browser but not in NodeJS.